code/upaya/bin/setup

```diff
@@ -14,6 +14,9 @@ Dir.chdir APP_ROOT do
 
   puts "== Installing dependencies =="
   run "gem install bundler --conservative"
+
+  run 'gem install foreman && gem update foreman'
```

Comment:
That's a great question. I don't know the actual reason, but the foreman README explicitly warns against adding it to the Gemfile: https://github.com/ddollar/foreman#installation